### ISSUE TITLE
is finished?hot to use?guy

### ISSUE THREAD

**Issue by caojianDavid** [joli80/RouterReboot#1]
the script is finished?
i want to use shell rebooting dir 655,i try to use ' wget reboot urlpath'  ,but login page use radom token for login. it 's different every time

**Comment by joli80** [comment]
Yes it is finished but I have not used in it a couple of years. When I used it it worked fine. It is possible D-link changed the login process in newer firmwares.

I made a blog post about it:
http://blog.twigg.se/2010/08/c-code-for-rebooting-dir-655-router.html
